subsys/net/lib/lwm2m/lwm2m_senml_cbor_types.h

```diff
@@ -18,6 +18,16 @@
 #include <string.h>
 #include "zcbor_common.h"
 
+enum lwm2m_senml_cbor_key {
```

Comment:
@oyvindronningstad It would be nice if the generator would create these automatically.